bin/tessel-2.js

```diff
@@ -252,6 +234,28 @@ module.exports = function(args) {
   parser.parse(args);
 };
 
+module.exports.closeSuccessfulCommand = function() {
+  process.exit(0);
+};
+
+// Allow options to be partially applied
+module.exports.closeFailedCommand = function(opts, err) {
```

Comment:
It's kind of ugly but I had to break `closeSuccessfulCommand` and `closeFailedCommand` into the module exports (and call them as `module.exports.X` elsewhere) to use them properly in tests. Other suggestions welcome.